programming_examples/shim_dma_2d/README.md

```diff
@@ -0,0 +1,35 @@
+# shim_dma_2d
+
+This example demonstrates how data may be moved using shim DMA operations. Specifically, in this example a 2-dimensional block of data (sometimes referred to in the code as an *image*) is set to have some specific values.
+
+It was transferred to a compute core using DMA. The compute core then modifies an upper corner of the data (sometimes referred to in code as the *tile*).
+
+The logic in this example is defined in [shim_dma_2d.py](shim_dma_2d.py).
+
+For illustrative purposes, there are 3 ways to run and test this example.
+
+## Run and test with AIR utility functions
```

Comment:
"Run and test with AIR compiler Python bindings"

Reply:
Fixed!